modules/telemetry/src/specific/planetssonification.cpp

```diff
@@ -0,0 +1,779 @@
+    _earthProperty.moonsEnabled.setValue(_earthProperty.toggleAll);
+}
+void PlanetsSonification::onEarthSettingChanged() {
+    sendPlanetData(EarthIndex);
+}
+
+// Mars
+void PlanetsSonification::onMarsAllChanged() {
+    _marsProperty.sizeDayEnabled.setValue(_marsProperty.toggleAll);
+    _marsProperty.gravityEnabled.setValue(_marsProperty.toggleAll);
+    _marsProperty.temperatureEnabled.setValue(_marsProperty.toggleAll);
+    _marsProperty.atmosphereEnabled.setValue(_marsProperty.toggleAll);
+    _marsProperty.moonsEnabled.setValue(_marsProperty.toggleAll);
+}
+void PlanetsSonification::onMarsSettingChanged() {
+    sendPlanetData(MarsIndex);
+}
+
+// Jupiter
+void PlanetsSonification::onJupiterAllChanged() {
+    _jupiterProperty.sizeDayEnabled.setValue(_jupiterProperty.toggleAll);
+    _jupiterProperty.gravityEnabled.setValue(_jupiterProperty.toggleAll);
+    _jupiterProperty.temperatureEnabled.setValue(_jupiterProperty.toggleAll);
+    _jupiterProperty.atmosphereEnabled.setValue(_jupiterProperty.toggleAll);
+    _jupiterProperty.moonsEnabled.setValue(_jupiterProperty.toggleAll);
+}
+void PlanetsSonification::onJupiterSettingChanged() {
+    sendPlanetData(JupiterIndex);
+}
+
+// Saturn
+void PlanetsSonification::onSaturnAllChanged() {
+    _saturnProperty.sizeDayEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.gravityEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.temperatureEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.atmosphereEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.moonsEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.ringsEnabled.setValue(_saturnProperty.toggleAll);
+}
+void PlanetsSonification::onSaturnSettingChanged() {
+    sendPlanetData(SaturnIndex);
+}
+
+// Uranus
+void PlanetsSonification::onUranusAllChanged() {
+    _uranusProperty.sizeDayEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.gravityEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.temperatureEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.atmosphereEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.moonsEnabled.setValue(_uranusProperty.toggleAll);
+}
+void PlanetsSonification::onUranusSettingChanged() {
+    sendPlanetData(UranusIndex);
+}
+
+// Neptune
+void PlanetsSonification::onNeptuneAllChanged() {
+    _neptuneProperty.sizeDayEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.gravityEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.temperatureEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.atmosphereEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.moonsEnabled.setValue(_neptuneProperty.toggleAll);
+}
+void PlanetsSonification::onNeptuneSettingChanged() {
+    sendPlanetData(NeptuneIndex);
+}
+
+bool PlanetsSonification::getData(const Camera* camera, int planetIndex,
```

Comment:
see previous comments 
```suggestion
bool PlanetsSonification::updateData(const Camera* camera, int planetIndex,
```